### DISCUSSION TITLE
Aren't there any plans to update the ISO image on the basis of Debian 12.8 this year?

### DISCUSSION THREAD

**Discussion by holton-jiang** [SpiralLinux/SpiralLinux-project#516] [General]
I hope it will support the latest kernel.

**Comment by geckolinux** [comment]
Hi there, Debian Stable never updates to a major new kernel version during the entire lifecycle of a Stable release. The newer kernel version in SpiralLinux comes from Debian Backports, which does not have any specific cycle for offering newer kernels, just as needed and if a developer has time to update it.
As for the other updates in the Debian Stable 12.x point releases, there are never new feature updates during a Stable release cycle, with the exception of Firefox and Thunderbird ESR, so a newer SpiralLinux ISO does not seem necessary at this time. Just install with the current SpiralLinux ISO and then update the installed system to the latest Debian packages.

**Reply by holton-jiang** [reply]
Understood, thanks for your reply.

**Comment by marwenius** [comment]
You can simply build a new ISO file yourself under Debian Bookworm(!), which then contains the latest packages at the time:
```
sudo apt update && sudo apt install live-build -y
wget -c https://github.com/SpiralLinux/SpiralLinux-project/raw/refs/heads/main/SpiralLinux-Cinnamon.tar.gz
tar xvf SpiralLinux-Cinnamon.tar.gz
cd SpiralLinux-Cinnamon
sudo ./spiral -i
```
(Using the Cinnamon edition as an example.)

It takes just under half an hour to build the ISO on my system (AMD Ryzen 5 3500C, 8GB RAM, 100 Mbps internet speed).

**Reply by holton-jiang** [reply]
Thank you,I'll give it a try.
